tasks/configure.yml

```diff
@@ -1,39 +1,53 @@
 ---
+
 ## ASSEMBLE CONFIG - FRONTEND
+
 - name: 'Create directory for the frontend'
-  file: path={{ etc_prefix }}/haproxy/frontends.d state=directory
+  file:
+    path: "{{ etc_prefix }}/haproxy/frontends.d"
+    state: directory
 
 - name: 'Empty the folder if not already empty'
   command: find {{ etc_prefix }}/haproxy/frontends.d -name *.cfg -exec rm -f {} \;
 
 - name: 'Build up the frontends'
-  template: src=frontend.cfg dest={{ etc_prefix }}/haproxy/frontends.d/{{ item.name }}.cfg
+  template:
+    src: "frontend.cfg"
```

Comment:
Is there a specific reason to not quote when we don't interpolate ?
I may sound a bit old school, but I like it when the code explicitly says : I intend to put a text string here. It has a delimiting effect that cannot be fooled by trailing spaces for instance, be it intentionally or unintentionally.
As the delimiters for variables `{{ }}` are rather explicit, I would not see a specific reason to quote them, as this, in turn, may seem redundant: `"{{ var }}"` doubly delimits `{{ var }}`
So unless the value has to be specifically interpreted (true, false, yes, no, ...), I prefer quoting.
Also, and I'm reaching to your next comment here, this is only the first pass of clean-up I am planning. I intend to actually switch some static values to default vars in the role to allow customization. This will transform some of the values into interpolated variables, that then will require quoting. I'm just overstepping a bit ;)
I may not be aware of some side effects of quoting however, so feel free to let me know !